geospaas_harvesting/harvesters.py

```diff
@@ -200,3 +199,16 @@ def _create_ingester(self):
             if parameter_name in self.config:
                 parameters[parameter_name] = self.config[parameter_name]
         return ingesters.CreodiasEOFinderIngester(**parameters)
+
+class LOCALHarvester(WebDirectoryHarvester):
+    """ Harvester class for some specific local files """
+    def _create_crawlers(self):
+        return [
+            crawlers.LocalDirectoryCrawler(
+                url,
+                includes = self.config.get('includes', None),
+                time_range = self.get_time_range()
+                )
+            for url in self.config['Addresses']
```

Comment:
Please use `paths` instead of `Addresses`, it's the proper term.

Reply:
Done in the next commit